RIPS/rip-7560.md

```diff
@@ -241,47 +242,54 @@ The amount of gas used by this frame is referred to as `senderValidationGasUsed`
 
 The frame must return 32 bytes `validationData` that is interpreted as:
 
-```solidity
-MAGIC_VALUE_SENDER <20 bytes>, validUntil <6 bytes>, validAfter <6 bytes>
-```
+- **authorizer** (20 bytes) - (magic value/address, defined below)
```

Comment:
Don't introduce the new term "authorizer" in this EIP. Just make it 20 bytes. We can add the explanation in the "Rationale" section.

Reply:
then how to call it? 20 byte magic?